plugin-viewport/src/test/kotlin/com/mapbox/maps/plugin/viewport/transition/TransitionUtilsTest.kt

```diff
@@ -0,0 +1,188 @@
+package com.mapbox.maps.plugin.viewport.transition
+
+import android.animation.Animator
+import android.animation.AnimatorSet
+import android.animation.ValueAnimator
+import io.mockk.mockk
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class TransitionUtilsTest {
+  @Test
+  fun `test normalizeBearing 1`() {
```

Comment:
seems `normalize Bearing 1-8` should be parameterized tests.
other can be moved to another test class

Reply:
This is a direct port of unit tests from nav sdk, I will keep it in the beta release and refactor these to parameterised tests before rc release.